.github/workflows/test_derper.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy Test Derper
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * *'
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+    
+env:
+    RUST_BACKTRACE: 1
+    RUSTFLAGS: -Dwarnings
+    RUSTDOCFLAGS: -Dwarnings
+    MSRV: "1.72"
+    SCCACHE_CACHE_SIZE: "50G"
+    
+jobs:
+    build_derper:
+        runs-on: [self-hosted, linux, X64]
+        if: github.ref_name=='main'
+        env:
+            # Using self-hosted runners so use local cache for sccache and
+            # not SCCACHE_GHA_ENABLED.
+            RUSTC_WRAPPER: "sccache"
+            RUST_BACKTRACE: full
+            RUSTV: stable
+        steps:
+        - uses: actions/checkout@v2
+        - name: Install rust
+          run: rustup toolchain install stable
```

Comment:
why choose this over the `dtolnay/rust-toolchain@stable` action we use everywhere else?

Reply:
Cause I copy and pasted it from the iroh ci.yml file 😢 
I'll update what I find.